src/cmd/init.rs

```diff
@@ -25,6 +26,8 @@ build_search_index = %SEARCH%
 # Put all your custom variables here
 "#;
 
+const LOCAL_UNC: &str = "\\\\?\\";
```

Comment:
can you add a comment about what this is? I will have forgotten about UNC in about a week

Reply:
Sure.
However, I saw the comments on the issue. Of course this is not the best fix, long term one will come as soon as the rust team will fix it properly. However in the mean time this is a simple workaround.
So if you want to merge this PR, I'll be happy to refine it. Otherwise you can just close it. I will not be disappointed, I did it to try to better understand the code so it is fine.

Reply:
The main thing is that I/we will never remember to check back on the Rust issue to see when it will be fixed :/